backend/services/implementations/schoolService.ts

```diff
@@ -18,6 +18,33 @@ class SchoolService implements ISchoolService {
     this.userService = userService;
   }
 
+  // This method gets all schools from the database.
+  async getAllSchools(): Promise<SchoolResponseDTO[]> {
```

Comment:
nit: use documentation format starting with `/**`

Reply:
just wondering - why do we add documentation to both the interface and implementation? is that a common practice you've seen?

Reply:
we can just add it to the interface - you don't have to do add it to the implementation but you can if you want to!